<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AreaLayoutFrame</class>
 <widget class="QWidget" name="AreaLayoutFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1064</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,1,0" columnstretch="1,1">
   <item row="2" column="1">
    <widget class="QTabWidget" name="tabAreaWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tbCtem">
      <attribute name="title">
       <string>CTEM</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="vCtemLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tbStem">
      <attribute name="title">
       <string>STEM</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="vStemLayout"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tbCbed">
      <attribute name="title">
       <string>CBED</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="vCbedLayout"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="tabPanel" name="tabGeneralWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::South</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="3">
        <widget class="QCheckBox" name="chkKeep">
         <property name="text">
          <string>Preserve areas on file change</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Resolution: </string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="cmbResolution">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>26</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>256</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>512</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>768</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1024</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1536</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2048</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3072</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4096</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8192</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="tabPanel" name="tabZWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::South</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tZ">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Slice thickness</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="EditUnitsBox" name="edtSliceThickness">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="7">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>43</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>44</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Slice offset</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="EditUnitsBox" name="edtSliceOffset">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Slice output count</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="EditUnitsBox" name="edtSliceOutput">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="chkSliceOutput">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="tabPanel" name="tabScaleWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::South</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tbScales">
      <attribute name="title">
       <string>Scales</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0,0,0,0,0,0">
       <item row="3" column="5">
        <widget class="QLabel" name="lblAngleScale">
         <property name="text">
          <string>--</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLabel" name="lblFreqScale">
         <property name="text">
          <string>--</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Reciprocal scale: </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="6">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="lblStemScaleX">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>--</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="lblRealScale">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>--</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QLabel" name="lblStemScaleY">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>--</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="5">
        <widget class="QLabel" name="lblAngleMax">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>--</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Reciprocal max: </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Real scale: </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="3">
        <widget class="QLabel" name="lblFreqMax">
         <property name="text">
          <string>--</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="5">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Angle Max: </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="5">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Angle scale: </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="lblStemXHeader">
         <property name="text">
          <string>STEM scale x:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLabel" name="lblStemYHeader">
         <property name="text">
          <string>STEM scale y</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="lblSlices">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>--</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Slices: </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0" rowspan="4">
    <widget class="tabPanel" name="tbStructurePlot">
     <property name="tabPosition">
      <enum>QTabWidget::South</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tPlot">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QVBoxLayout" name="vPlotLayout">
         <item>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="2">
            <widget class="QLabel" name="lblStructStartX">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>--</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>x range: </string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QLabel" name="lblStructFinishX">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>--</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <spacer name="horizontalSpacer_10">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string> to </string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <spacer name="horizontalSpacer_9">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="lblStructStartY">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>--</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string> to </string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>y range:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QLabel" name="lblStructFinishY">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>--</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,1,0">
           <item>
            <spacer name="horizontalSpacer_8">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="chkShowRect">
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <property name="text">
              <string>Show boxes</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>View: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cmbViewDirection">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>26</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>Top</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Front</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Left</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Bottom</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Back</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Right</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btnApplyUpdate">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>26</height>
              </size>
             </property>
             <property name="text">
              <string>Apply &amp;&amp; Update</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EditUnitsBox</class>
   <extends>QLineEdit</extends>
   <header>controls/editunitsbox.h</header>
  </customwidget>
  <customwidget>
   <class>tabPanel</class>
   <extends>QTabWidget</extends>
   <header>controls/tabpanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tbStructurePlot</tabstop>
  <tabstop>chkShowRect</tabstop>
  <tabstop>cmbViewDirection</tabstop>
  <tabstop>btnApplyUpdate</tabstop>
  <tabstop>tabGeneralWidget</tabstop>
  <tabstop>cmbResolution</tabstop>
  <tabstop>chkKeep</tabstop>
  <tabstop>tabZWidget</tabstop>
  <tabstop>edtSliceThickness</tabstop>
  <tabstop>edtSliceOffset</tabstop>
  <tabstop>chkSliceOutput</tabstop>
  <tabstop>edtSliceOutput</tabstop>
  <tabstop>tabAreaWidget</tabstop>
  <tabstop>tabScaleWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
